public/js/themes.js

```diff
@@ -0,0 +1,48 @@
+// Managing theme changes:
+const darkId = 'dark-theme-css'
+const lightId = 'light-theme-css'
+const darkEl = document.querySelector(`link[id="${darkId}"]`)
+const lightEl = document.querySelector(`link[id="${lightId}"]`)
+
+function changeCssToDark(toDark) {
+  if (toDark) {
+    darkEl.media = ''
+    lightEl.media = 'none'
+  }
+  else {
+    lightEl.media = ''
+    darkEl.media = 'none'
+  }
+}
```

Comment:
I like the idea of DRY-ing up the two functions by moving the two selectors in the upper scope, but I'd avoid the flag argument in this case, here is a good explanation on that.

https://martinfowler.com/bliki/FlagArgument.html